dashboard/test/models/user_test.rb

```diff
@@ -574,6 +574,28 @@ class UserTest < ActiveSupport::TestCase
     assert student.reload.encrypted_password != old_password
   end
 
+  test 'user in_progress_and_completed_scripts does not include deleted scripts' do
+    user = create :user
+    real_script = Script.starwars_script
+    fake_script = create :script
+
+    user_script_1 = create :user_script, user: user, script: real_script
+    user_script_2 = create :user_script, user: user, script: fake_script
```

Comment:
I assume I don't need to do manual cleanup of records created via FactoryGirl?  It doesn't look like we do elsewhere.